### PR TITLE
17.2.8 Release notes

### PR DESCRIPTION
**Pull request by yuriw**

### CODE REVIEW

File: src/en/news/blog/2024/v17-2-8-quincy-released/index.md [hunk 1]
```diff
@@ -0,0 +1,1452 @@
...
+  - "release"
+  - "quincy"
+---
+
+This is the eighth backport release in the Quincy series. We recommend all users update to this release.
+
+## Notable Changes
+
+- RADOS: `get_pool_is_selfmanaged_snaps_mode` C++ API has been deprecated
+  due to being prone to false negative results.  It's safer replacement is
```
Comment (zdover23) [inline src/en/news/blog/2024/v17-2-8-quincy-released/index.md:15]: s/It's/Its/

File: src/en/news/blog/2024/v17-2-8-quincy-released/index.md [hunk 2]
```diff
@@ -0,0 +1,1452 @@
...
+  `pool_is_in_selfmanaged_snaps_mode`.
+- RBD: When diffing against the beginning of time (`fromsnapname == NULL`) in
+  fast-diff mode (`whole_object == true` with `fast-diff` image feature enabled
```
Comment (zdover23) [inline src/en/news/blog/2024/v17-2-8-quincy-released/index.md:18]: s/with `fast-diff`/with the `fast-diff`/

File: src/en/news/blog/2024/v17-2-8-quincy-released/index.md [hunk 3]
```diff
@@ -0,0 +1,1452 @@
...
+  and valid), diff-iterate is now guaranteed to execute locally if exclusive
+  lock is available.  This brings a dramatic performance improvement for QEMU
+  live disk synchronization and backup use cases.
+- RBD: The option ``--image-id`` has been added to `rbd children` CLI command,
```
Comment (zdover23) [inline src/en/news/blog/2024/v17-2-8-quincy-released/index.md:22]: s/to `rbd-children`/to the `rbd-children`/

File: src/en/news/blog/2024/v17-2-8-quincy-released/index.md [hunk 4]
```diff
@@ -0,0 +1,1452 @@
...
+tags:
+  - "release"
+  - "quincy"
+---
+
+This is the eighth backport release in the Quincy series. We recommend all users update to this release.
+
+## Notable Changes
+
+- RADOS: `get_pool_is_selfmanaged_snaps_mode` C++ API has been deprecated
```
Comment (zdover23) [inline src/en/news/blog/2024/v17-2-8-quincy-released/index.md:14]: s/`get_pool_is_selfmanaged_snaps_mode` C++ API/The `get_pool_is_selfmanaged_snaps_mode` C++ API/

File: src/en/news/blog/2024/v17-2-8-quincy-released/index.md [hunk 5]
```diff
@@ -0,0 +1,1452 @@
...
+  due to being prone to false negative results.  It's safer replacement is
+  `pool_is_in_selfmanaged_snaps_mode`.
+- RBD: When diffing against the beginning of time (`fromsnapname == NULL`) in
+  fast-diff mode (`whole_object == true` with `fast-diff` image feature enabled
+  and valid), diff-iterate is now guaranteed to execute locally if exclusive
```
Comment (zdover23) [inline src/en/news/blog/2024/v17-2-8-quincy-released/index.md:19]: s/if exclusive/if an exclusive/

File: src/en/news/blog/2024/v17-2-8-quincy-released/index.md [hunk 6]
```diff
@@ -0,0 +1,1452 @@
...
+  lock is available.  This brings a dramatic performance improvement for QEMU
+  live disk synchronization and backup use cases.
+- RBD: The option ``--image-id`` has been added to `rbd children` CLI command,
+  so it can be run for images in the trash.
+- RBD: `RBD_IMAGE_OPTION_CLONE_FORMAT` option has been exposed in Python
```
Comment (zdover23) [inline src/en/news/blog/2024/v17-2-8-quincy-released/index.md:24]: s/`RBD_IMAGE_OPTION_CLONE_FORMAT` option/The `RBD_IMAGE_OPTION_CLONE_FORMAT` option/

File: src/en/news/blog/2024/v17-2-8-quincy-released/index.md [hunk 7]
```diff
@@ -0,0 +1,1452 @@
...
+  bindings via `clone_format` optional parameter to `clone`, `deep_copy` and
```
Comment (zdover23) [inline src/en/news/blog/2024/v17-2-8-quincy-released/index.md:25]: s/via `clone_format`/via the `clone_format`/

s/to `clone`/to the `clone`/

File: src/en/news/blog/2024/v17-2-8-quincy-released/index.md [hunk 8]
```diff
@@ -0,0 +1,1452 @@
...
+  `migration_prepare` methods.
+- RBD: `RBD_IMAGE_OPTION_FLATTEN` option has been exposed in Python bindings via
```
Comment (zdover23) [inline src/en/news/blog/2024/v17-2-8-quincy-released/index.md:27]: s/`RBD_IMAGE_OPTION _FLATTEN` option/The`RBD_IMAGE_OPTION_FLATTEN` option/

File: src/en/news/blog/2024/v17-2-8-quincy-released/index.md [hunk 9]
```diff
@@ -0,0 +1,1452 @@
...
+  `flatten` optional parameter to `deep_copy` and `migration_prepare` methods.
```
Comment (zdover23) [inline src/en/news/blog/2024/v17-2-8-quincy-released/index.md:28]: s/`flatten`/the `flatten/

s/to `deep_copy`/to the `deep_copy`/

**Review by zdover23 — CHANGES_REQUESTED** [review]
@yuriw: I've made some suggestions for the addition of articles to sentences. 

If you tell me that it's okay to push to this branch, I'll take care of these alterations myself. I just wanted to make sure that you had agreed to it before I acted.

**Comment by yuriw** [comment]
@zdover23 sure

**Review by zdover23 — APPROVED** [review]

**Comment by yuriw** [comment]
https://wip-yuriw-release-17.2.8-post-main.ceph.io/en/news/blog/2024/v17-2-8-quincy-released/

**Review by ljflores — APPROVED** [review]

**Pull request merged**